setup_build_env.sh

```diff
@@ -10,6 +10,7 @@ cp -r $SOURCE_DIR/thirdparty $BUILD_DIR
 cd $BUILD_DIR
 ./thirdparty/download_thirdparty.sh
 ./thirdparty/build_thirdparty.sh
+./thirdparty/versions.sh
```

Comment:
This doesn't work for me. I think build_thirdparty leaves the working directory in build/.